packages/webpack-config-anansi/src/node.js

```diff
@@ -2,6 +2,9 @@ import path from 'path';
 import nodeExternals from 'webpack-node-externals';
 import StatsPlugin from 'stats-webpack-plugin';
 
+// this should be the keys of alias from base/index.js
+const NODE_POLYFILLS = ['stream', 'path', 'crypto', 'buffer', 'vm'];
```

Comment:
there's a *lot* more core modules than this that bundlers polyfill - `process`, for one - making `fs` into an empty object, and a number of other things. I believe https://github.com/webpack/node-libs-browser/blob/master/index.js is a more complete list.

Reply:
My initial thought was to just copy nextjs's selection as I assumed it would match all the libraries used in npm, but I suppose it's nearly impossible to truly know and there's no harm in being more comprehensive, so I'll add the complete list. Thanks for the input!